src/arch/host/include/arch/task.h

```diff
@@ -52,7 +52,7 @@ static inline int arch_allocate_tasks(void)
  * \brief Runs task.
  * \param[in,out] task Task data.
  */
-static inline int arch_run_task(struct task *task)
+static inline int arch_run_task(struct edf_task *task)
```

Comment:
task not edf_task - this must be unified.

Reply:
Done